.circleci/config.yml.j2

```diff
@@ -171,25 +175,43 @@ workflows:
               only:
                 - master
                 - slack-build-approvals
-{%- endfor %}
-{%- endfor %}
-{%- if image_map.keys() | dev_releases | length > 0 %}
+      {%- endfor %}{# distribution in distributions #}
+      {%- endif %}{# not edge_build #}
+      {%- endfor %}{# ac_version, distributions in image_map.items() #}
+  {%- if image_map.keys() | dev_releases | length > 0 %}
   nightly:
     when:
       and:
         - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
         - equal: [ "every-midnight-utc", << pipeline.schedule.name >> ]
     jobs:
-{%- for ac_version, distributions in image_map.items() %}
-{%- set airflow_version = ac_version | get_airflow_version %}
-{%- if "dev" in ac_version and airflow_version not in dev_allowlist  %}
-{%- for distribution in distributions %}
+      {%- for ac_version, distributions in image_map.items() %}
+      {%- set airflow_version = ac_version | get_airflow_version %}
+      {%- set edge_build = ac_version | is_edge_build -%}
+      {%- if "dev" in ac_version and airflow_version not in dev_allowlist  %}
+
+      {%- if edge_build %}
+      - download-file:
+          name: download-latest-{{ airflow_version | replace('.dev', '') }}-build
+          url: https://pip.astronomer.io/simple/astronomer-certified/latest-{{ airflow_version | replace('.dev', '') }}.build
+          file: latest-{{ airflow_version | replace('.dev', '') }}.build
+      {%- endif %}
+
+      {%- for distribution in distributions %}
       - build:
           name: build-{{ airflow_version }}-{{ distribution }}
+          {%- if edge_build %}
+          requires:
+            - download-latest-{{ airflow_version | replace('.dev', '') }}-build
+          {%- endif %}
           airflow_version: {{ airflow_version }}
           distribution_name: {{ distribution }}
           dev_build: true
-          extra_args: "--build-arg VERSION=$(curl https://pip.astronomer.io/simple/astronomer-certified/latest-{{ airflow_version | replace('.dev', '') }}.build)"
+          extra_args:
+            {%- if edge_build %}
+            --tag $(cat latest-{{ airflow_version | replace('.dev', '') }}.build)
```

Comment:
Will this tag be pushed?

https://github.com/astronomer/ap-airflow/blob/master/.circleci/config.yml.j2#L550-L601